test/haartest.cpp

```diff
@@ -0,0 +1,20 @@
+#include "gtest/gtest.h"
+#include <haarfeature.h>
+#include <vector>
+#include <iostream>
+
+TEST(CreateHaarFeature, CanCreateHaarFeature) {
+  std::vector<std::vector<int>> testImg(50, std::vector<int>(50));
+  IntegralImage img(testImg);
+  HaarFeature* haar0 = createDefault();
+  HaarFeature* haar1 = createSpecific(img, Point(2,3), 1, 2, false);
+  HaarFeature* haar2 = createRandom(img);
+  EXPECT_EQ(haar0->getPoint().getX(), 0);
+  EXPECT_EQ(haar0->getPoint().getY(), 0);
+  int a = haar2->getType();
+  auto b = haar1->getPoint();
+  auto c = haar2->getSize();
+  std::cout << img.getImgWidth() << ", type = " << a << "\n";
```

Comment:
Not good implementation of test. Usually `Expect_EQ` at the last line of a `TEST`.